pkg/descheduler/controllers/migration/controller.go

```diff
@@ -108,7 +100,26 @@ func New(args runtime.Object, handle framework.Handle) (framework.Plugin, error)
 		return nil, err
 	}
 
-	if err = c.Watch(&source.Kind{Type: &sev1alpha1.PodMigrationJob{}}, &handler.EnqueueRequestForObject{}, &predicate.Funcs{
+	// New filter
+	filter, arbitrationFilter, err := arbitrator.NewFilter(controllerArgs, handle)
```

Comment:
What is the difference between filter and arbitrationFilter?

Reply:
I just expose different interface to `Arbitrator` and `Migration Controller`.

Reply:
Maybe, I should expose `filter struct`.